test/main.js

```diff
@@ -23,14 +21,10 @@ function init(callback) {
 	require('./../app');
 
 	// Run the battle engine in the main process to keep our sanity
-	let BattleEngine = global.BattleEngine = require('./../battle-engine');
 	for (let listener of process.listeners('message')) {
 		process.removeListener('message', listener);
 	}
```

Comment:
I believe this code can be retired, too.

It originally intended to revert a ``.process.on('message')`` side-effect from loading the battle engine, but it's no longer needed since a more sane battle IPC model was implemented at f6a7c4bd305